transaction/src/manifest/generator.rs

```diff
@@ -56,14 +59,10 @@ use radix_engine_interface::types::GlobalAddress;
 use radix_engine_interface::types::InternalAddress;
 use radix_engine_interface::types::ResourceAddress;
 use radix_engine_interface::*;
-use sbor::rust::borrow::Borrow;
-use sbor::rust::collections::IndexMap;
-use sbor::rust::str::FromStr;
-use sbor::rust::vec;
-use sbor::*;
+use sbor::prelude::*;
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub enum GeneratorError {
+pub enum GeneratorErrorKind {
```

Comment:
Side refactor: In the below three, `expected_type` should be `expected_value_kind`

Reply:
Done